Translation/en/Text/Main/Menu-FreeH.txt

```diff
@@ -83,9 +83,9 @@ H段階=H-Level
 
 //Time of Day
 時間帯=Time Of Day
-朝=Morning
+朝=AM
 昼=Noon
-夕方=Evening
+夕方=PM
```

Comment:
Dusk

Reply:
Ok, will change to Dusk. If I have to change this one more time I'll let as twilight!